### DISCUSSION TITLE
Removing the unnecessary warning when default exception mapper is disabled

### DISCUSSION THREAD

**Discussion by moghaddam** [resteasy/resteasy-microprofile#46] [General]
When the default exception mapper in MicroProfile Rest Client is disabled by setting the `microprofile.rest.client.disable.default.mapper=true`, a warning appears in the log for every single calls to `RestClientBuilderImpl.build()`. The log is added in https://github.com/resteasy/resteasy/pull/2850 and you can see that there is also a comment by reviewer about reducing the log level for this message to `debug`, but it got merged anyway.

I was wondering if it should really be considered as warning :thinking: 

We have recently upgraded to WildFly 26 and there are tons of such warnings in the log while it's an expected behavior in our case. I think others could have had a similar issue too.

**Comment by jamezp** [comment]
This does look like it should be reduced. I think even a debug is overkill as it shouldn't be logged that many times IMO.

As a temporary workaround you could disable the logger for that specific category since luckily it appears to have it's own. With CLI it would be something like:
```
/subsystem=logging/logger=org.jboss.resteasy.microprofile.client.ExceptionMapping.HandlerException:add(level=ERROR)
```

**Reply by moghaddam** [reply]
Yes. That's exactly what I did. Would it be ok if I send a PR for that to remove the log?

**Reply by jamezp** [reply]
Yes that would be great. After looking at it again I do think a `debug` message makes sense though.

**Reply by moghaddam** [reply]
I just sent the #49 PR for this.
